R/calc_creat.R

```diff
@@ -2,39 +2,98 @@
 #'
 #' Calculate an estimated serum creatinine. Function takes vectorized input as well.
 #'
-#' Uses equations described in Ceriotti et al. Clin Chem. 2008, and Junge W et al.
-#' Clin Chim Acta. 2004. For age 15-18, a linear interpolation is used between
-#' equations for <15 and >18 years as described in Johanssen A et al. Ther Drug
-#' Monit 2011.
+#' For children and adolescents: Uses equations described in Ceriotti et al. 
+#' Clin Chem. 2008, and Junge W et al. Clin Chim Acta. 2004. For age 15-18, 
+#' a linear interpolation is used between equations for <15 Johanssen A et al. 
+#' Ther Drug Monit 2011.
+#' For adults:  Two methods are available Johanssen et al. Ther Drug Monit 2011.
+#' which describes a flat serum creatinine dependent on sex, and Brooks et al 
+#' (unpublished) which is a linear regression built on the NHANES open-source
+#' data and takes into account sex, age, weight, and height. Equation: lm(SCr~Age+Sex+Weight+Height, data=NHANES)
```

Comment:
Nitpick, for proper rendering in help:
```suggestion
#' data and takes into account sex, age, weight, and height. 
Equation: `lm(SCr~Age+Sex+Weight+Height, data=NHANES)`
```